index.html

```diff
@@ -165,7 +165,7 @@ <h3>Develop tools</h3>
       <h3>Write Fortran software</h3>
       <p>
       Or just write Fortran software for your research, business, or schoolwork.
-      You can learn how to get started <a href="/tutorial/">here</a>.
+      You can learn how to get started <a href="/learn/">here</a>.
```

Comment:
```suggestion
      You can learn how to get started <a href="{{site.baseurl}}/learn/">here</a>.
```

Reply:
Oops, I am really sorry I already merged this. I think we have to submit a new PR.

Reply:
That's an improvement, it should work without that as well.

Reply:
Thanks for the PR @aktech. We should keep to the habit of prepending all internal site links like this. See [here](https://github.com/fortran-lang/fortran-lang.org/blob/master/CONTRIBUTING.md#internal-site-links) in the (incomplete) contributor document.

Reply:
Makes sense, thanks for the link. Will keep that in mind.